uvicorn/protocols/http/httptools_impl.py

```diff
@@ -66,21 +91,23 @@ def __init__(self, config, server_state, _loop=None):
         self.default_headers = server_state.default_headers
 
         # Per-connection state
-        self.transport = None
-        self.flow = None
-        self.server = None
-        self.client = None
-        self.scheme = None
-        self.pipeline = deque()
+        self.transport: asyncio.Transport = None  # type: ignore[assignment]
+        self.flow: FlowControl = None  # type: ignore[assignment]
```

Comment:
Are you using the `# type: ignore[assignment]` just to avoid some `assert self.transport is not None`s later?

Reply:
Yes